EventsExpress/ClientApp/src/containers/event-item-view.js

```diff
@@ -33,28 +35,31 @@ class EventItemViewWrapper extends Component{
 
     render(){   
         const { isPending } = this.props.event;
-  
         return isPending
             ? <Spinner />
-            : <EventItemView 
+            : <EventItemView
+                inventories={this.props.inventories} 
                 event={this.props.event}
                 match={this.props.match} 
                 onLeave={this.onLeave} 
                 onJoin={this.onJoin}
                 onCancel={this.onCancel}
                 onApprove={this.onApprove}
-                current_user={this.props.current_user} 
+                current_user={this.props.current_user}
+                get_inventories_by_event_id={this.props.get_inventories_by_event_id} 
```

Comment:
Why do you need to pass this method from here if your component is already connected to state? Consider moving it there.